_posts/2023-10-11-14.0.0-release.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: "Apache Arrow 14.0.0 Release"
+date: "2023-10-11 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+
+The Apache Arrow team is pleased to announce the 14.0.0 release. This covers
+over 3 months of development work and includes [**XXX resolved issues**][1]
+from [**YYY distinct contributors**][2]. See the [Install Page](https://arrow.apache.org/install/)
+to learn how to get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Community
+
+Since the 13.0.0 release, Metehan Yildirim and Oleks V. have been invited to be committers.
```

Comment:
@kou @alamb I don't think there are new PMCs in the last 3 months and there are only 2 new committers, could you confirm this is correct, please?

Reply:
Correct!

Reply:
According to the board report:

```
Apache Arrow was founded 2016-01-19 (8 years ago)
There are currently 99 committers and 50 PMC members in this project.
The Committer-to-PMC ratio is roughly 7:4.

Community changes, past quarter:
- No new PMC members. Last addition was Dewey Dunnington on 2023-06-22.
- Oleks V. was added as committer on 2023-09-30
- Metehan Yildirim was added as committer on 2023-08-29
```